code/datums/powernet_data.dm

```diff
@@ -8,6 +8,13 @@
 	/// associated list of various data fields
 	var/list/fields = list(  )
 
+proc/FindRecordByField(var/list/datum/data/record/L, var/F, var/V)
```

Comment:
Could you give `F` and `V` slightly more descriptive names please?

Reply:
How about field and value?

Reply:
yes